app/models/qc_results_upload_factory.rb

```diff
@@ -41,8 +41,10 @@ def create_entities!
 
   # @param [Object] CSV row e.g. { col_header_1: row_1_col_1 }
   def create_data(row_object)
-    # 1. Always create Long Read QC Decision
-    lr_qc_decison = create_qc_decision!(row_object[LR_DECISION_FIELD], :long_read)
+    # 1. If required, create Long Read QC Decision
+    if row_object[LR_DECISION_FIELD]
```

Comment:
There is a bit of duplication here. Now we are processing both decision types in the same way they could maybe be combined?

Reply:
I've pulled it out it to its own method, but agree some refactoring could be done. Had a quick try but it wasn't coming to me so will leave it for now and get it into UAT, but will keep pondering on it

Reply:
OK. I had an idea of how to do it but it can wait.